cmd/olympus/actions/app.go

```diff
@@ -127,6 +129,26 @@ func App(config *AppConfig) (*buffalo.App, error) {
 }
 
 func getWorker(store storage.Backend, eLog eventlog.Eventlog) (worker.Worker, error) {
+	workerType := env.OlympusBackgroundWorkerType()
+	switch workerType {
+	case "redis":
+		return registerRedis(store, eLog)
+	case "memory":
+		return registerInMem(store, eLog)
+	default:
+		return nil, fmt.Errorf("Provided worker type %s not one of redis|memory", workerType)
```

Comment:
let's put some sane default here, no need to be this restricted. Log warning and default to inmem as it does not require deps to be setup.

Reply:
Made the change to in-memory worker for default, now prints a message using the standard log package. Thanks for the review.